### DISCUSSION TITLE
How to get all parent/child id?

### DISCUSSION THREAD

**Discussion by san-123** [GrapesJS/grapesjs#3944] [General]
Hi @artf ,

 I need to fetch all ParentI/child ID's like dv1, dv2, ul3, pl check below, Is it possible?

```
<Div id="dv1"> 
<Div id ="dv2">
 <Ul I'd="ul3>

 <Li>test content</li>

 </Ul> 
</Div> 
<Div>
```

 Pl help Thanks

**Comment by ronaldohoch** [comment]
You can do this:
`editor.Canvas.getDocument().body.querySelectorAll("[id]")`

![image](https://user-images.githubusercontent.com/2287371/142435380-d8f1d4cb-512f-4886-a226-83fc906df2f9.png)

**Reply by san-123** [reply]
Thanks a lot . It's working
